packages/storybook/stories/components/input/3_demo.stories.ts

```diff
@@ -1,5 +1,5 @@
 import { ODS_THEME_COLOR_INTENTS } from '@ovhcloud/ods-common-theming';
-import { ODS_INPUT_SIZES, ODS_INPUT_TYPES } from '@ovhcloud/ods-components/input';
+import { ODS_COMMON_FIELD_SIZES, ODS_COMMON_INPUT_TYPES } from '@ovhcloud/ods-common-core';
```

Comment:
should be `import { ODS_INPUT_SIZES, ODS_INPUT_TYPES } from '@ovhcloud/ods-components/input';` here ;)
the story behave the same as a user who would use the input component (so it won't know about common-core)